tests/ci/cdk/cdk/codebuild/github_ci_integration_omnibus.yaml

```diff
@@ -250,3 +250,35 @@ batch:
         image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-x86:ubuntu-22.04_gcc-12x_latest
         variables:
           AWS_LC_CI_TARGET: "tests/ci/integration/run_ntp_integration.sh"
+
+    - identifier: openldap_integration_x86_64
+      buildspec: tests/ci/codebuild/common/run_simple_target.yml
+      env:
+        type: LINUX_CONTAINER
+        privileged-mode: false
+        compute-type: BUILD_GENERAL1_MEDIUM
+        image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-x86:ubuntu-22.04_gcc-12x_latest
+        variables:
+          AWS_LC_CI_TARGET: "tests/ci/integration/run_openldap_integration.sh"
+          OPENLDAP_BRANCH: "master"
+          OPENLDAP_RELEASE: "OPENLDAP_REL_ENG_2_5"
```

Comment:
I wonder whether by removing [the quotes here](https://github.com/aws/aws-lc/blob/0e38c147b184a1904f166ffcb1cf47f8828c91b3/tests/ci/codebuild/common/run_simple_target.yml#L13) we would be able to pass arguments directly to the script, instead of setting environment variables?
```
phases:
  build:
    commands:
      - ./${AWS_LC_CI_TARGET}
```

Reply:
I'll try this out. I was trying to avoid clumping the arguments to `AWS_LC_CI_TARGET` since it's not very neat